srv/isc-agent/plugins/tcp/http/models.py

```diff
@@ -1,177 +1,95 @@
-from sqlalchemy import create_engine
-from sqlalchemy import Column, Integer, Text, BLOB
+from sqlalchemy import create_engine, Column, Integer, Text, JSON, ForeignKey
 from sqlalchemy.orm import registry
 
 mapper_registry = registry()
 
 Base = mapper_registry.generate_base()
 
 
-# Sigs model
-class Sigs(Base):
-    __tablename__ = 'sigs'
+class Signature(Base):
+    __tablename__ = 'signature'
 
     id = Column(Integer, primary_key=True)
-    # Should this be Text vs String?
     pattern_description = Column(Text)
     pattern_string = Column(Text)
     db_ref = Column(Text)
     module = Column(Text)
 
     def __repr__(self):
-        return f'''Sigs(id={self.id!r}, patternDescription={self.patternDescription!r},
-        patternString={self.patternString!r}, db_ref={self.db_ref!r}, module={self.module!r} )'''
+        return f"{self.__class__.__name__}({self.id}: {self})"
 
+    def __str__(self):
+        return self.pattern_string
 
-class HdrResponses(Base):
-    __tablename__ = 'hdr_responses'
+
+class HeaderResponse(Base):
+    __tablename__ = 'header_responses'
 
     id = Column(Integer, primary_key=True)
     sig_id = Column(Integer)
     header_field = Column(Text)
```

Comment:
Let's remove the `_field` reference. We know its a field. Same for anywhere else using field.